.gitignore

```diff
@@ -30,3 +30,4 @@ testgen/.gradle
 out/
 
 OpenApiDocsTest.java
+/example/api-docs.json
```

Comment:
please remove, build dir is excluded already

Reply:
done, now the file is saved in the build directory so no entry in the gitignore is needed